tools/tf2openapi/tfmodel/tfsignaturedef.go

```diff
@@ -0,0 +1,27 @@
+package tfmodel
+
+/**
+TFSignatureDef defines the signature of supported computations in the TensorFlow graph, including their inputs and
+outputs. It is the internal model representation for the SignatureDef defined in the TensorFlow repository
+[tensorflow/core/protobuf/meta_graph.proto]
+*/
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+type TFSignatureDef struct {
+	Name    string
+	Inputs  [] TFTensor
+	Outputs [] TFTensor
+}
+
+
+func (sigDef *TFSignatureDef) Schema() *openapi3.Schema {
+	// TODO add options for single tensor
+	multiTensorSchema := *openapi3.NewObjectSchema().WithProperties(make(map[string]*openapi3.Schema))
```

Comment:
Better to not do deference assignments. Deference at the last moment (especially since you're re-pointering again).